packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -118,7 +131,15 @@ export function encodePubKey(value: string) {
 
   return b58cencode(value.substring(2, 42), prefix.KT);
 }
-
+/**
+ *
+ * @description Base58 encode an address without predefined prefix
+ * @param value Address to base58 encode (tz4) hex dec
+ * @returns return address
+ */
+export function encodePubKeyL2Address(value: string) {
```

Comment:
Sorry for being picky, but because tz4 is a public key hash, is it confusing to have publicKey in the method name?